tox.ini

```diff
@@ -5,15 +5,22 @@
 
 [tox]
 envlist = py27
-          py35
-
 
 [testenv]
-passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
-commands =
-    coverage run --source=toto setup.py test
-    coveralls
+setenv =
+    SIMPLE_SETTINGS=toto.settings
+    # PYTHONPATH = {toxinidir}
 
 deps =
-   -rrequirements.txt
-   coveralls
+    -rrequirements.txt
+    pycrypto
+    cryptography
+    pytest
+    PyNaCl
+    coverage
```

Comment:
maybe we could create a file called requirements-ci.txt and add these there. Just keep -rrequirements-ci.txt...